src/6.6/6.6.2.0.md

```diff
@@ -0,0 +1,59 @@
+---
+nav:
+  title: v6.6.2.0
+meta:
+  date: "2024-00-00"
+---
+# Release notes Shopware 6.6.2.0
+
+## Abstract
+
+
+
+## System requirements
+
+* tested on PHP 8.2 and 8.3
+* tested on MySQL 8 and MariaDB 10.11
+
+## Improvements
+
+### Typescript support for webpack config in extensions
+Extension can now use typescript (.ts and .cts files) for their webpack configuration. 
+
+### Added meteor component library to the administration
+
+The [Meteor component library](https://shopware.design/meteor-components/) was added as a new dependecy to the administration.
+All components from the library are now also directly available in the administration.
+
+### Sync option for CLI theme commands
+
+The `theme:compile` and `theme:change ` command now accept `--sync` option to compile themes synchronously. The `--sync` option is useful for CI/CD pipelines, when at runtime themes should be compiled async, but during the build process you want sync generation.
+
+### Order Approval improvements
```

Comment:
@kermie this feature is part of the commercial plugin in the b2b context, should we add a note or something about that?

Reply:
Yep. Would do like this:

### Order Approval improvements (valid for commercial plugin in B2B context)